release/nightly_tests/dataset/multi_node_train_benchmark.py

```diff
@@ -47,10 +58,19 @@ def parse_args():
         type=int,
         help="Read the input dataset n times, used to increase the total data size.",
     )
+    parser.add_argument(
+        "--target-worker-gb",
+        default=4,
+        type=int,
+        help=(
+            "Number of GB per worker for selecting a subset "
+            "from default dataset. -1 means the whole dataset"
+        ),
+    )
     parser.add_argument(
         "--read-task-cpus",
         default=1,
-        type=int,
+        type=float,
```

Comment:
Shall we scale the target max block size by this value?

Reply:
@stephanie-wang which max block size are you referring to here? and what would be an appropriate scaling factor?

Reply:
DataContext.target_max_block_size.

You can just multiply it by this value to scale it properly.